.gitmodules

```diff
@@ -10,6 +10,9 @@
 [submodule "org.lflang/src/lib/cpp/reactor-cpp"]
 	path = org.lflang/src/lib/cpp/reactor-cpp
 	url = https://github.com/lf-lang/reactor-cpp
+[submodule "org.lflang/src/lib/platform/arduino/Arduino-CMake-Toolchain"]
+	path = org.lflang/src/lib/platform/arduino/Arduino-CMake-Toolchain
+	url = https://github.com/a9183756-gh/Arduino-CMake-Toolchain.git
```

Comment:
This repository should probably be moved to the lf-lang org.

Reply:
Marten requested to keep it as a submodule under the new platform lib directory in case we add more arduino libraries down the line.

Reply:
Right. My point was that we should move this submodule repo into the lf-lang org (like reactor-c).

Reply:
Would a fork of the original repository suffice? It hasn't been updated March 2020.

Reply:
Resolved.

Reply:
@Soroosh129 why do we need to create a fork and reference that? We are not the maintainers of that submodule.

Reply:
Right, I didn't know that when I made my comment. @arengarajan99 clarified that for me.